retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_update_product]
+// Update product from a catalog using Retail API
+
+using Google.Cloud.Retail.V2;
+using System;
+using System.Linq;
+
+/// <summary>
+/// The update product sample class.
+/// </summary>
+public class UpdateProductSample
+{
+    private static readonly Random Random = new Random();
+    private static readonly string GeneratedProductId = RandomAlphanumericString(14);
+
+    /// <summary>
+    /// Generate the random alphanumeric string.
+    /// </summary>
+    /// <param name="length">The required length of alphanumeric string.</param>
+    /// <returns>Generated alphanumeric string.</returns>
+    private static string RandomAlphanumericString(int length)
```

Comment:
I think you can substitute all of this for
```suggestion
Guid.NewGuid().ToString("N").Substring(0, 14).
```
Here and everywhere.

Reply:
Agree, applied suggested improvement.